docs/build/dapp-staking/dapp_staking_architecture.md

```diff
@@ -0,0 +1,672 @@
+---
+sidebar_position: 1
+title: V3 Technical Architecture
+---
+
+## Introduction
+
+:::important
+Please note that this is a technical document on the design of the dApp Staking V3 as it has been imagined. It serves as a reference and helps to better understand its architecture. 
+
+:::
+
+For the final implementation on Astar Network or Shiden, please refer to the documentations on Astar Github.
+
+**Resources:**   
+https://docs.astar.network/docs/learn/dapp-staking/dapp-staking-protocol
+https://github.com/AstarNetwork/Astar/tree/master/pallets/dapp-staking-v3
+https://github.com/AstarNetwork/Astar/tree/master/precompiles/dapp-staking-v3
+
+# I. Overview
+
+### Eras & Periods
+
+The concept of eras is kept from the old dApp staking and a new concept of `periods` is introduced.
+
+Each `period` consists of two `subperiods` - `voting subperiod` followed by a `build&earn subperiod`:
+- `voting subperiod` length is expressed in eras, but it only lasts for a single era:
+    - **E.g.** if standard era length is 7200 blocks, and voting period lasts for 7 standard eras, it means that voting period lasts for 7200 x 7 = 50400 blocks. For the sake of simplicity, this is considered to be a single voting period era.
+- `build&earn subperiod` length is also expressed in eras but unlike `voting period`, can consist of 1 or more standard era lenghts:
+    - **E.g.** if it is said that build&earn subperiod lasts for 30 eras, it means that 30 distinct eras will happen during that period, before a new voting period is triggered.
+
+Rewards are only earned from eras that are part of build&earn period.
```

Comment:
I believe this whole subchapter can be omitted since it's already covered by existing docs (this is a duplicate).